### PR TITLE
Bump composer version

### PR DESCRIPTION
**Pull request by aspfohl**
I don't have a reason to prefer 18.1 > 18.0, but maybe @mvpatel2000 @dakinggg you'd have opinions https://github.com/mosaicml/composer/releases/tag/v0.18.1

### CODE REVIEW

**Comment by dakinggg** [comment]
@aspfohl it should be 0.18.1. Also can you please kick off a regression testing run?

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
 ]
 
 install_requires = [
-    'mosaicml[libcloud,wandb,oci,gcs]>=0.17.2,<0.18',
+    'mosaicml[libcloud,wandb,oci,gcs]>=0.18.1,<0.19',
```
Comment (dakinggg) [inline setup.py:53]: there are a few more places you need to bump the version in the extra deps

**Comment by aspfohl** [comment]
will pause for now 👍

File: setup.py (hunk 1 above)
Comment (aspfohl) [reply setup.py:53]: yep, caught in the test failures 👍

**Pull request closed without merging**